.github/workflows/publish-to-live-pypi.yml

```diff
@@ -9,12 +9,14 @@ jobs:
   build-n-publish:
     name: Build and publish Python 🐍 distributions 📦 to pypi
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```

Comment:
@fsbraun These are missing the environment section which pypi uses to verify the project matches up to where it'll be published;

https://github.com/django-cms/django-filer/blob/master/.github/workflows/publish-to-live-pypi.yml#L12

Reply:
🤦‍♂️